include/abb_librws/rws_poco_client.h

```diff
@@ -400,14 +402,9 @@ class POCOClient
   void extractAndStoreCookie(const std::string cookie_string);
 
   /**
-   * \brief Static constant for the default timeout for HTTP requests, in microseconds.
-   */
-  static const Poco::Int64 DEFAULT_TIMEOUT = 400000;
-  
-  /**
-   * \brief Static constant for a long timeout for HTTP requests, in microseconds.
+   * \brief Static constant for the default HTTP communication timeout [microseconds].
    */
-  static const Poco::Int64 LONG_TIMEOUT = 10000000;
+  static const Poco::Int64 DEFAULT_HTTP_TIMEOUT = 400000;
```

Comment:
```suggestion
  static const Poco::Int64 DEFAULT_HTTP_TIMEOUT = 400e3;
```

Also: that's quite a reduction (10 sec to less than 0.5 sec)?

Reply:
The `LONG_TIMEOUT` was previously only used by default by the RWS subscription functionality, and now it's using the `DEFAULT_SUBSCRIPTION_TIMEOUT` instead (which is 40 seconds).